remount.py

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/env python3
+import argparse
+import subprocess
+import os
+from sys import exit
+from tempfile import mkdtemp
+from typing import Dict, Tuple
+from pathlib import Path
+
+def determine_platform(mountpoint: str) -> str:
+    pkgs = [
+        subprocess.run(['chroot', str(mount_path), 'pacman', '-Qqs', 'odroid']).check_output().strip(),
+        subprocess.run(['chroot', str(mount_path), 'pacman', '-Qqs', 'linux-raspberrypi']).check_output.strip(),
+    ]
+    
+    if "uboot-odroid" in pkgs:
```

Comment:
It would be more pythonic to use a dict here.

```python
PKG_KERNEL_MAP: Dict[str,str] = {
    "uboot-odroid": "odroid",
    "linux-raspberrypi: "rpi3",
    etc.
}

for k, v in MAP.items():
    if k in pkgs:
        return v
return "odroid"
```